knative-serving/config-applications/pvcs-for-serving.adoc

```diff
@@ -4,8 +4,9 @@ include::_attributes/common-attributes.adoc[]
 = Persistent Volume Claims for Serving
 :context: pvcs-for-serving
 
-Some serverless applications need permanent data storage. 
-To achieve this, you can configure persistent volume claims (PVCs) for your Knative services.
+Some serverless applications need permanent data storage. You can provide data storage for Knative services by configuring different volume types such as `secret`, `configMap`, `projected`, and `emptyDir`. 
+
+To achieve this, you can configure persistent volume claims (PVCs) for your Knative services. The PersistentVolume types are implemented as plugins. Serverless applications also support PersistentVolume types such as `awsElasticBlockStore`, `gcePersistentDisk`, and `cephfs`.
```

Comment:
PVCs are just another type of volume source, so let's follow the way we phrased it upstream. We recently merged the new [storage documentation](https://knative.dev/development/serving/services/storage/).

> You can provide data storage for Knative Services by configuring different volumes types. Serving supports mounting the [volume types](https://kubernetes.io/docs/concepts/storage/volumes): emptyDir, secret, configMap and projected. [PersistentVolumes](https://kubernetes.io/docs/concepts/storage/persistent-volumes/) are supported but require a [feature flag](https://knative.dev/development/serving/configuration/feature-flags/) to be enabled.

QE needs to verify this:

> `awsElasticBlockStore`, `gcePersistentDisk`, and `cephfs`

Reply:
Sure! I will try to include this.

Reply:
@skonto I have a doubt. Our Section title is **Persistent Volume Claims for Serving** so we need to mention about PVCs as well, right? 

Also, we can not include the upstream doc links in the downstream documents. For example, [PersistentVolumes](https://kubernetes.io/docs/concepts/storage/persistent-volumes/) or [volume types](https://kubernetes.io/docs/concepts/storage/volumes/)